components/collection/HeroButtons.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div>
+    <Loader v-model="isLoading" :can-cancel="false" />
```

Comment:
Loader in HeroButtons?
a bit strange

Reply:
If there is no loader once the owner clicks the delete button nothing happens. It will lead to bad UX, I guess

Reply:
i get the need for the Loader
just think it is more suitable to be at the page level and not in a sub-component

up to your consideration, if you find it is not worth the hustle then I let it go

Reply:
ahh I see. agree on that, would be better if I can call something like global loader

Reply:
yeah.
I had a similar situation with the shopping cart, where the loader could be triggered from anywhere in the app

that's why I created the <Buy/> component

Reply:
Temporarily, I moved it to the layout. It would be good if we could put it somewhere globally